lcls2-cc-lib/Library/Library.plcproj

```diff
@@ -137,13 +153,15 @@
     <PlaceholderResolution Include="LCLS General">
       <Resolution>LCLS General, * (SLAC)</Resolution>
     </PlaceholderResolution>
-    <PlaceholderResolution Include="lcls-twincat-motion">
-      <Resolution>lcls-twincat-motion, * (SLAC)</Resolution>
-    </PlaceholderResolution>
     <PlaceholderResolution Include="PMPS">
       <Resolution>PMPS, * (SLAC - LCLS)</Resolution>
     </PlaceholderResolution>
   </ItemGroup>
+  <ItemGroup>
+    <LibraryReference Include="lcls-twincat-motion,1.3.2,SLAC">
```

Comment:
Can't merge into lib if you pin the version. Libraries need to stay unpinned to the project that uses the library can decide the version.

Reply:
Makes sense, good to know for libraries within libraries. Resolved in commit 739e16d